pkg/proxy/iptables/proxier.go

```diff
@@ -950,15 +949,6 @@ func (proxier *Proxier) syncProxyRules() {
 	// Accumulate NAT chains to keep.
 	activeNATChains := map[utiliptables.Chain]bool{} // use a map as a set
 
-	// We are creating those slices ones here to avoid memory reallocations
-	// in every loop. Note that reuse the memory, instead of doing:
-	//   slice = <some new slice>
-	// you should always do one of the below:
```

Comment:
is this memory reallocation no longer a problem so we can remove it?

Reply:
https://pkg.go.dev/sync#example-Pool

Reply:
That seems to be for avoiding allocations you are doing directly. I don't think there's any way that could be used to optimize append()s

Reply:
yeah, I realized later that this is about slices .... that is much trickier ... 
anyway, this guy seem that studied this topic, https://github.com/xdg-go/zzz-slice-recycling , just I found it interesting and I wanted to share it, not related to this PR